src/proc/colorizer.cpp

```diff
@@ -136,7 +136,13 @@ namespace librealsense
 
     colorizer::colorizer()
         : colorizer("Depth Visualization")
-    {}
+    {
+        std::string device_name = this->get_info(RS2_CAMERA_INFO_NAME);
```

Comment:
IMO, this is too much code to insert to each place we want to increment a counter.
I expected to see 1 line here.
Can we add the implementation inside the `aus` and call a service function each place we use it?

Reply:
fixed

Reply:
I don't see the new code..